docs/source/testing/larva/options.rst

```diff
@@ -0,0 +1,265 @@
+    </Result>
+
+Using ignore
+----------------------
+
+**ignoreRegularExpressionKey**
+
+This replaces all occurrences of regular expression with IGNORE. By writing  
+
+.. code-block::
+
+  ignoreRegularExpressionKey1.key="^\d+$"
+
+all the content of this current result will be ignored, so no matter what’s in it, the current result will always pass the test.
+
+**ignoreKey**
+
+This replaces all matching occurrences of provided key with IGNORE. By writing
+
+.. code-block::
+
+  ignoreKey1.key=RecordID
+
+all 4 “RecordID” which appeared in the above example will be replaced with IGNORE. In this case, comparing RecordID will still fail, because it won’t replace the content of RecordID.
+
+**ignoreContentBetweenKeys**
+
+This replace all characters found between key1 and key2 with IGNORE. By writing
+
+.. code-block::
+
+  ignoreContentBetweenKeys1.key1=<RecordID>
+  ignoreContentBetweenKeys1.key2=</RecordID>
+
+content between those two tags will be replaced with IGNORE, so comparing RecordID will always succeed no matter what the value is of it.
+
+And by writing 
+
+.. code-block::
+
+  ignoreContentBetweenKeys1.key1=RecordID='
+  ignoreContentBetweenKeys1.key2='
+
+this will replace the RecordID value in the element <Message> with IGNORE.
+
+**ignoreKeysAndContentBetweenKeys**
+
+This is similar to ignoreContentBetweenKeys, the only difference is that it replaces all characters found between key1 and key2, including key1 and key2, with IGNORE. Write as 
+
+.. code-block::
+
+  ignoreKeysAndContentBetweenKeys1.key1=<RecordID>
+  ignoreKeysAndContentBetweenKeys1.key2=</RecordID>
+
+
+**ignoreContentBeforeKey**
+
+This replaces everything before provided key has been found with IGNORE. By writing
+
+.. code-block::
+
+  ignoreContentBeforeKey1.key=ABC
+
+content before “ABC”, in this case is “<RecordID> 12</RecordID> <Category>”, will be replaced with IGNORE.
+
+**ignoreContentAfterKey**
+
+The opposite of ignoreContentBeforeKey is ignoreContentAfterKey, this replaces everything after  provided key has been found with IGNORE. By writing 
+
+.. code-block::
+
+  ignoreContentAfterKey1.key=ABC
+
+it will replace everything after “ABC” with IGNORE.
+
+**ignoreCurrentTimeBetweenKeys**
+
+This replaces time found between key1 and key2 with IGNORE_CURRENT_TIME, pattern can be null. Write as
+
+.. code-block::
+
+  ignoreCurrentTimeBetweenKeys1.key1=<Timestamp>
+  ignoreCurrentTimeBetweenKeys1.key2=</Timestamp>
+  ignoreCurrentTimeBetweenKeys1.pattern=yyyy-MM-dd'T'HH:mm:ss.SSSZ
+  ignoreCurrentTimeBetweenKeys1.margin=12345
+  ignoreCurrentTimeBetweenKeys1.errorMessageOnRemainingString=true/false
+
+
+There is an other way of using ignore: ``step.service.read=ignore``
+this is available since IAF 7.6 version, it allows ignoring the result of a specific test step. For example, in the ``scenario.properties`` file, you write ``step7.database.Generic.read=ignore`` to ignore the result of this step.
```

Comment:
This paragraph has another subject than the preceeding ones have. Can you give it its own heading?

Reply:
I checked that this issue has been resolved. Thanks.